terraform/s3.tf

```diff
@@ -0,0 +1,4 @@
+resource "aws_s3_bucket" "aspen-data" {
+  bucket = join("", ["aspen-data-", var.DEPLOYMENT_ENVIRONMENT])
```

Comment:
is this bucket going to be holding data relating to certain db fields (like `s3_bucket` fields in the db)? if so i think a more specific name like `aspen-db-data` might be better, just in case we have other aspen data stored on s3 later on? `aspen-app-data` might also be a good name.

Reply:
Renamed to `aspen-db-data-*`.

I hand-modified the db entries in the staging db, and am copying the data to the new bucket...